src/openassetio-python/tests/cmodule/resources/_openassetio_test/errorsTest.cpp

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2023 The Foundry Visionmongers Ltd
+/**
+ * Bindings used for testing errors behaviour.
+ * Specifically, the conversion from cpp -> python.
+ */
+
+#include <pybind11/pybind11.h>
+
+#include <openassetio/errors/exceptions.hpp>
+namespace py = pybind11;
+
```

Comment:
[plan] Can we add some documentation on the who/what/when/where/why

Reply:
Addressed as part of #1104